src/components/vacation-requests-table/vacation-requests-table-toolbar/toolbar-form-toggle-button.tsx

```diff
@@ -0,0 +1,48 @@
+import { Variant } from "@mui/material/styles/createTypography";
+import { ComponentType, ReactNode } from "react";
+import { ButtonIconProps } from "../../../types";
+import { Button, ButtonOwnProps, Typography } from "@mui/material";
+
+/**
+ * Component properties
+ */
+interface Props {
+  children?: ReactNode;
+  buttonVariant?: ButtonOwnProps["variant"];
+  titleVariant?: Variant;
+  value: boolean;
+  setValue: (value: boolean) => void;
+  ButtonIcon?: ComponentType<ButtonIconProps>;
+  title?: string;
+}
+
+/**
+ * Generic Toggle Button component
+ *
+ * @param props component properties
+ */
+const FormToggleButton = ({
+  children,
+  value,
+  setValue,
+  ButtonIcon,
+  title,
+  titleVariant,
+  buttonVariant
+}: Props) => (
+  <Button
+    variant={buttonVariant ? buttonVariant : "contained"}
+    onClick={() => {
+      setValue(!value);
+    }}
+    sx={{
+      width: "100%"
+    }}
+  >
+    {ButtonIcon && <ButtonIcon />}
+    {title && <Typography variant={titleVariant ? titleVariant : "h6"}>&nbsp;{title}</Typography>}
```

Comment:
Same here, can you style with some margin/ padding rather than use the nbsp?

Reply:
Removed non-breaking space and added marginLeft to Typography:
`      <Typography variant={titleVariant ? titleVariant : "h6"} marginLeft={1}>
        {title}
      </Typography>`